### PR TITLE
Datavizcolors

### PR DESCRIPTION
**Pull request by HerrTopi**
Closes: INSTUI-4174, INSTUI-4175, INSTUI-4176, INSTUI-4177

### CODE REVIEW

File: packages/__docs__/globals.js [hunk 1]
```diff
@@ -82,7 +86,10 @@ const globals = {
   useEffect,
   useState,
   useRef,
-  ThemeColors
+  additionalPrimitives,
+  datavisualization,
```
Comment (joyenjoyer) [inline packages/__docs__/globals.js:90]: Consider camelCase for this as it is seen in other names.

Comment (HerrTopi) [reply packages/__docs__/globals.js]: Nice catch, updated

**Review by matyasf — CHANGES_REQUESTED** [review]

**Review by matyasf — CHANGES_REQUESTED** [review]

File: docs/guides/color-system.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: Color System
+category: Guides
+order: 5
+---
+
```
[the remark this answers is not in the document]
Comment (HerrTopi) [reply docs/guides/color-system.md:6]: Fixed

File: packages/canvas-high-contrast-theme/src/index.ts [hunk 3]
```diff
@@ -41,16 +39,7 @@ const {
 
 const key = 'canvas-high-contrast'
 
-export type CanvasHighContrastTheme = BaseTheme & {
-  key: 'canvas-high-contrast'
-} & typeof canvasHighContrast
```
[the remark this answers is not in the document]
Comment (HerrTopi) [reply packages/canvas-high-contrast-theme/src/index.ts:46]: Nice catch, done

File: packages/ui-themes/README.md [hunk 4]
```diff
@@ -21,15 +21,15 @@ npm install @instructure/ui-themes
 - global theming:
 
   ```js
-  import { canvas } from '@instructure/ui-themes'
+  import canvas from '@instructure/ui-themes'
```
[the remark this answers is not in the document]
Comment (HerrTopi) [reply packages/ui-themes/README.md:24]: Nice catch, done

**Review by matyasf — APPROVED** [review]

**Pull request merged**